### PR TITLE
Update README.md to include instructions for required missing steps

### PR DESCRIPTION
**Pull request by dmurphy18**
Updated the README.md to add additional steps that I found were missing from the instructions in order to successfully access the vCenter

### CODE REVIEW

**Review by joechainz — APPROVED** [review]
Looks good to me!

File: README.md [hunk 1]
```diff
@@ -54,20 +54,52 @@ The [Salt Contributing guide][salt-contributing] has a lot of relevant informati
...
+      config_dir: local/etc/salt
+    EOF
+
+    # 6. Create a pillar file for you configuration
+    cat << EOF> local/srv/my_vsphere_conf.sls
+    # vCenter
+    vmware_config:
+      host: insert_host_name_or_ip_address
+      password: insert_password
+      user: insert_user_name
```
Comment (waynew) [inline README.md:89]: **suggestion** Use TEST-NET IP or example.com domain. Also update to use our modern key.

```suggestion
    saltext.vmware:
        # Or use IP address, e.g. 203.0.113.42
        host: vsphere.example.com
        password: CorrectHorseBatteryStaple
        user: BobbyTables
```

File: README.md [hunk 2]
```diff
@@ -54,20 +54,52 @@ The [Salt Contributing guide][salt-contributing] has a lot of relevant informati
...
+
+    # 7. Create a pillar top file
+    cat << EOF>  local/srv/pillar.sls
+    base:
+      saltdev:
+        - my_vsphere_conf
     EOF
 
-    # 3. Create a test config file:
+    # 8. Create a test config file:
```
Comment (waynew) [inline README.md:99]: **suggestion** let's rewrite this:

```suggestion
    # 8. (deprecated but not removed yet) If you're contributing to the project and need to run the tests, create a test config file:
```

File: README.md [hunk 3]
```diff
@@ -54,20 +54,52 @@ The [Salt Contributing guide][salt-contributing] has a lot of relevant informati
...
     python tools/test_value_scraper.py -c local/vcenter.conf
 
-    # 4. Create a test config file for VMC:
+    # 8. Create a test config file for VMC:
```
Comment (waynew) [inline README.md:102]: **suggestion** same as test config file - deprecated but not removed yet.

**Review by waynew — CHANGES_REQUESTED** [review]
Couple of changes - spotted a typo, and also prepping for changes that are on the horizon.

**Pull request merged**